packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1191,11 +1194,30 @@ enum ResizeImagePolicy {
 /// Instructs Flutter to decode the image at the specified dimensions
 /// instead of at its native size.
 ///
+/// The [width] and [height] parameters refer to logical pixels.
+///
+/// Logical pixels have roughly the same visual size across devices, whereas
+/// physical pixels correspond to actual device hardware.
+/// The number of physical pixels per logical pixel is described by the [ui.FlutterView.devicePixelRatio].
```

Comment:
Could be nice to keep that line from going over:

```suggestion
/// The number of physical pixels per logical pixel is described by the
/// [ui.FlutterView.devicePixelRatio].
```